tests/Datagrid/ProxyQueryTest.php

```diff
@@ -212,4 +214,30 @@ public function testSetHint()
 
         $this->assertEquals(2, $result[0]['id']);
     }
+
+    public function testAddOrderedColumnsCompositeId()
+    {
+        $qb = $this->em->createQueryBuilder()
+                       ->select('IDENTITY(o.id1) as id1, IDENTITY(o.id2) as id2')
+                       ->distinct()
+                       ->from(self::COMPOSITE_ID_CLASS, 'o')
+                       ->orderBy('o.id1', 'ASC')
+                       ->addOrderBy('o.id2', 'ASC');
+
+        $pq = $this->getMockBuilder('Sonata\DoctrineORMAdminBundle\Datagrid\ProxyQuery')
```

Comment:
Please use the `::class` notation everywhere you can

Reply:
@greg0ire ready for review